breathing_cat/build.py

```diff
@@ -436,6 +436,117 @@ def _search_for_general_documentation(
     return general_documentation
 
 
+def _copy_readme(source_dir: Path, destination_dir: Path) -> str:
+    """Searches for a README in the source and copies it to the destination directory .
+
+    Searches for a "README.md" or "README.rst" (case-insensitive) in the source
```

Comment:
We should also look for "README" and assume markdown

Reply:
Make sense but I think it will need a bit of rewrite.  I'll create an issue to add this in a separate PR.

Reply:
#7